docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -20,401 +51,388 @@ The scenarios of defining placement rules in SQL include:
 - Increase the replica count of more important data
 - Separate irrelevant data into different stores to improve availability
 
-## Define placement rules
+These scenarios usually fit into one of the following categories:
 
-There are 3 kinds of operations on the placement:
+1. An optimization or availability use case (such as improving data access locality)
+2. A compliance use case (such as ensuring data resides within a geographic region)
 
-* ADD: Add more replicas for one role.
-* ALTER: Override the replica configuration for one role.
-* DROP: Remove the replica configuration for one role.
+This proposal makes some intentional decisions so that both categories of use-cases can be handled correctly. It improves upon the earlier proposal by reducing the risk of misconfiguration which can affect compliance use cases.
 
-They’re all achieved by executing `ALTER TABLE` statements.
+## Detailed Design
 
-### Add placement rules
+### New Syntax Overview
 
-Adding new replicas can be done by one or more `ADD PLACEMENT POLICY` clauses:
+There are two ways to specify placement rules:
 
-```sql
-ALTER TABLE table_name
-	ADD PLACEMENT POLICY CONSTRAINTS=constraints ROLE=role REPLICAS=replicas,
-	...
-```
+1. By assigning placement directly on a database, table or partition (direct assignment)
+2. By creating a new `PLACEMENT POLICY` and then applying the placement policy to a database, table or partition (placement policy)
+
+Using a `PLACEMENT POLICY` will be recommended for compliance requirements, since it can allow administrators to better keep track of usage. This can be seen as similar to how complex environments will use `ROLES` for management instead of directly assigning privileges to users.
 
-This statement indicates TiDB to add replicas for all data of table `table_name`, including indexes.
+Both syntaxes are considered [`table_option`](https://dev.mysql.com/doc/refman/8.0/en/alter-table.html)s, and available in both `CREATE TABLE` and `ALTER TABLE` contexts.
```

Comment:
Customers may use the 'pd-ctl' to set placement rule, what is the expected behavior if the placement rule that is set by 'pd-ctl' is inconsistent with SQL?

Reply:
Discussed with zhangming before, and the result is that we don't care. You can't prevent customers from inserting invalid rules. And it will take effect immediately despite TiDB have or don't have a mechanism to synchronize rules from persistent storage to PD.